test/ci_workflow_gen.go

```diff
@@ -116,13 +112,12 @@ var (
 		"xb_recovery",
 		"mysql80",
 		"vreplication_across_db_versions",
-		"vreplication_multicell",
-		"vreplication_cellalias",
 		"vreplication_basic",
+		"vreplication_cellalias",
```

Comment:
This one could also be merged IMO, but doesn't have to be here.

Reply:
We can do this later if the current set of workflows runs without flakiness.